scif/main/commands.py

```diff
@@ -174,15 +174,18 @@ def run(self, app=None, args=None):
     # Cut out early if the app doesn't have a runscript
     config = self.app(app)
     if 'apprun' not in config:
-        bot.exit('%s does not have a runscript.' % app)
+        bot.debug('%s does not have a runscript.' % app)
```

Comment:
Would this pattern work here? 

```python
interactive = False
if "apprun" not in config:
    bot.debug('%s does not have a runscript.' % app)
    interactive = True
```

Reply:
Yes.  I originally wrote it that way, but later felt that it was preferable to be more explicit so I changed it.

Either is fine with me.  Let me know if you'd like me to change it.

Reply:
Yes please! Then we will be ready for merge, and I'll update and ready for release.